crates/ruff/src/args.rs

```diff
@@ -506,6 +508,9 @@ pub struct FormatCommand {
     pub range: Option<FormatRange>,
 }
 
+#[derive(Clone, Debug, clap::Parser)]
+pub struct ServerCommand;
```

Comment:
I don't have a strong opinion on the name. One advantage of using `lsp` over `server` is that we could add `ruff server start` and `ruff server stop`  commands in the future if we support a daemon. The LSP spec also has some recommendations about what arguments should be supported that we might not want to have on a generic `server` command. However, we could probably also do this by using `ruff server` with no sub-command to connect to the LSP directly`.

Reply:
If we end up supporting a language server _and_ a daemon, we should probably call the daemon something other than 'server', in my opinion.

Here's why I think `server` is a good name for this command:
* It fits our command naming scheme better, since none of our other commands use acronyms
* It's more immediately clear as to what it's doing, especially if someone isn't familiar with the term "LSP".